ports/servoshell/prefs.rs

```diff
@@ -747,14 +743,6 @@ fn print_debug_options_usage(app: &str) {
         "dump-stacking-context-tree",
         "Print the stacking context tree after each layout.",
     );
-    print_option(
-        "dump-display-list",
```

Comment:
This one still works. Is this being removed intentionally? Only the `dump-display-list-json` option was removed in #34994.